src/tests/ftest/util/daos_core_base.py

```diff
@@ -34,42 +35,85 @@ class DaosCoreBase(TestWithServers):
     :avocado: recursive
     """
 
+    TEST_PATH = "/run/daos_tests/Tests/*"
+
     def __init__(self, *args, **kwargs):
         """Initialize the DaosCoreBase object."""
         super(DaosCoreBase, self).__init__(*args, **kwargs)
         self.subtest_name = None
 
-        test_timeout = self.params.get("test_timeout",
-                                       '/run/daos_tests/Tests/*')
+        test_timeout = self.params.get("test_timeout", self.TEST_PATH)
         if test_timeout:
             self.timeout = test_timeout
 
     def setUp(self):
         """Set up before each test."""
-        self.subtest_name = self.params.get("test_name",
-                                            '/run/daos_tests/Tests/*')
+        self.subtest_name = self.params.get("test_name", self.TEST_PATH)
         self.subtest_name = self.subtest_name.replace(" ", "_")
+
         # obtain separate logs
         self.update_log_file_names(self.subtest_name)
+
         super(DaosCoreBase, self).setUp()
 
+    @fail_on(CommandFailure)
+    def start_server_managers(self):
+        """Start the daos_server processes on each specified list of hosts.
+
+        Enable scalable endpoint if requested with a test-specific
+        'scalable_endpoint' yaml parameter.
+        """
+        # Enable scalable endpoint (if requested) prior to starting the servers
+        scalable_endpoint = self.params.get("scalable_endpoint", self.TEST_PATH)
+        if scalable_endpoint:
+            for manager in self.server_managers:
+                # CRT_CTX_SHARE_ADDR=1
+                manager.set_config_value("crt_ctx_share_addr", 1)
+                # CRT_CREDIT_EP_CTX=0
+                # CRT_CTX_SHARE_ADDR=1
+                # CRT_CTX_NUM=8
+                env_vars_list = manager.get_config_vale("env_vars")
+                for env_vars in env_vars_list:
+                    env_vars["CRT_CREDIT_EP_CTX"] = 0
```

Comment:
not sure why ep credits are being set, this envariable has no relationship to whether scalable enpoints is enabled or not

Reply:
It came from [this](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/server_utils_params.py#L334) comment, which is apparently inaccurate.